recipes/opentelemac/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "opentelemac" %}
+{% set version = "v8p4r1" %}
```

Comment:
please update to `v8p5r0` and upload the archive on your [github repo](https://github.com/nicogodet/otm-sources/)

Reply:
While thinking back to it, it would be better to use archive from https://gitlab.pam-retd.fr/otm/telemac-mascaret